src/server/index.ts

```diff
@@ -124,5 +125,13 @@ export async function createServer(routeConfig?: RouteConfig) {
   await server.register(pluginErrorPages)
   await server.register(blipp)
 
+  server.state('cookieConsent', {
+    ttl: 365 * 24 * 60 * 60 * 1000, // 1 year in ms
+    clearInvalid: true,
+    isSecure: config.get('isProduction'),
+    path: '/',
+    encoding: 'none' // handle this inside the application so we can share frontend/backend cookie modification
```

Comment:
Can probably drop `'none'` as it's [one of the defaults](https://github.com/hapijs/statehood/blob/master/lib/index.js#L45-L55)

```suggestion
    path: '/'
```

Reply:
I'll leave it just for the comment, makes it explict.